tests/fuzz/src/read_network-fuzzer.cc

```diff
@@ -38,7 +38,9 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
     try {
         InferenceEngine::Core ie;
         InferenceEngine::CNNNetwork network = ie.ReadNetwork(net, weights_blob);
-    } catch (const InferenceEngine::details::InferenceEngineException& error) {
+    } catch (const InferenceEngine::details::InferenceEngineException&) {
```

Comment:
Not required if it has not any specific handling. `std::exception` is base class for it.